docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_label: "Configure OIDC"
+title: "Configure OIDC"
+description: "Learn how to configure OIDC so Palette displays the Virtual Machine Dashboard."
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 15
+tags: ["vmo", "oidc"]
+---
+
+
+
+
+Palette displays the Virtual Machine dashboard based on the OpenID Connect (OIDC) Identity Provider option that you select in the Kubernetes layer of the infrastructure profile.
+
+
+## Prerequisites
+
+- A configured infrastructure profile. For more information, review [Create a Cluster Profile](/cluster-profiles/task-define-profile).
```

Comment:
This URL needs to point to the file name.

Reply:
Fixed.